Mathlib/Data/List/Lookmap.lean

```diff
@@ -82,17 +81,21 @@ open Perm in
 theorem perm_lookmap (f : α → Option α) {l₁ l₂ : List α}
     (H : Pairwise (fun a b => ∀ c ∈ f a, ∀ d ∈ f b, a = b ∧ c = d) l₁) (p : l₁ ~ l₂) :
     lookmap f l₁ ~ lookmap f l₂ := by
-  induction' p with a l₁ l₂ p IH a b l l₁ l₂ l₃ p₁ _ IH₁ IH₂; · simp
-  · cases h : f a
```

Comment:
I'm glad this is getting fixed. I think this is a particularly egregious example of the lack of explicit structure in `induction'`